src/app/app-routing.module.ts

```diff
@@ -5,27 +5,19 @@ import { CallbackComponent } from './components/callback/callback.component'
 import { PlaylistComponent } from './components/playlist/playlist.component'
 import { SettingsComponent } from './components/settings/settings.component'
 import { AuthGuardService } from './services/auth-guard/auth-guard.service';
+import { PublicComponent } from './layouts/public/public.component'
+import { SecureComponent } from './layouts/secure/secure.component'
+import { PUBLIC_ROUTES } from './public.routes'
+import { SECURE_ROUTES } from './secure.routes'
```

Comment:
All routes that need an authenticated user need to be put in the `secure.routes` file.